src/pages/home/report/ReportActionCompose.js

```diff
@@ -116,6 +119,14 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+/**
+  * Return amount of emoji. For small screen we can fit 3 items and for large we show up to 5 items
```

Comment:
```suggestion
  * Return number of emoji suggestions. For small screen we can fit 3 items and for large we show up to 5 items
```

Reply:
I am not sure if this should be a part of `ReportActionCompose`. Should this be moved to `EmojiUtils`?